### PR TITLE
Remove remaining MSDN link in article

### PR DESCRIPTION
**Pull request by nemrism**
Replace msdn link in how-to-use-platform-invoke-to-play-a-wave-file.md
Contributes to #2037

### CODE REVIEW

**Review by rpetrusha — APPROVED** [review]
Thanks for updating the link, @nemrism. I do have one request for a change that is unrelated to your PR. If you'd prefer, we can make the change.

**Comment by nemrism** [comment]
@rpetrusha, do you mean that you'll handle this update in another PR?

**Comment by rpetrusha** [comment]
Either in this PR or a new PR,, @nemrism.

File: docs/csharp/programming-guide/interop/how-to-use-platform-invoke-to-play-a-wave-file.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ ms.assetid: f7f62f53-e026-4c40-b221-3a26adb0c2c5
...
+ This example code uses `DllImport` to import `winmm.dll`'s `PlaySound` method entry point as `Form1 PlaySound()`. The example has a simple Windows Form with a button. Clicking the button opens a standard windows <xref:System.Windows.Forms.OpenFileDialog> dialog box so that you can open a file to play. When a wave file is selected, it is played by using the `PlaySound()` method of the winmm.DLL assembly method. For more information about winmm.dll's `PlaySound` method, see [Using the PlaySound function with Waveform-Audio Files](https://docs.microsoft.com/windows/desktop/multimedia/using-playsound-to-play-waveform-audio-files). Browse and select a file that has a .wav extension, and then click **Open** to play the wave file by using platform invoke. A text box shows the full path of the file selected.
```
Comment (rpetrusha) [inline docs/csharp/programming-guide/interop/how-to-use-platform-invoke-to-play-a-wave-file.md]: Sorry, @nemrism. My comment was evidently lost. This is unrelated to your PR, but it's a very noticeable error in the original text. winmm.dll is neither an assembly nor a method. Could you change "of the winmm.DLL assembly method" to read "in the winmm.dll assembly"?

Comment (nemrism) [reply docs/csharp/programming-guide/interop/how-to-use-platform-invoke-to-play-a-wave-file.md]: As `winmm.dll` isn't an assembly nor a method,  should we rather change it to read "of the `winmm.dll` library"?

Comment (rpetrusha) [reply docs/csharp/programming-guide/interop/how-to-use-platform-invoke-to-play-a-wave-file.md]: Either "in" of "of" is fine.

**Comment by rpetrusha** [comment]
I apologize, @nemrism. I'd meant to merge this three days ago. I'll merge it now, and it should be live on docs.microsoft.com in the next day or two.

**Pull request merged**